coops-ui/src/components/AnnualReport/AGMDate.vue

```diff
@@ -82,7 +82,7 @@ import { FormType } from '@/interfaces'
     ...mapGetters(['lastFilingDate'])
   }
 })
-export default class AGMDate extends Mixins(DateMixin) {
+export default class AgmDate extends Mixins(DateMixin) {
```

Comment:
FYI - I changed this so we could use the recommended markup for this component (`<agm-date ...>`). If I had left it as AGMDate then Vue would have expected "a-g-m-date" :/

Reply:
PS I did not change the filename because git gets confused by that.

Reply:
got it, thanks and I like the consistent naming. nice to keep lifting the code forward.